script/set_version.sh

```diff
@@ -2,15 +2,15 @@
 
 set -e
 
-if [ "$#" -ne 1 ]; then
-    echo "usage: $0 version"
+if [ "$#" -lt 1 ] || [ "$#" -gt 2 ]; then
+    echo "usage: $0 version [image_name]"
     exit 1
 fi
 
 location=$(dirname $0)
 version=$1
 image_name=${2:-docker.io\/apache\/camel-k}
-sanitized_image_name=${image_name/\//\\\/}
```

Comment:
are we sure about this?

Reply:
Found it on stackoverflow :smile: : https://stackoverflow.com/questions/13210880/replace-one-substring-for-another-string-in-shell-script

Reply:
it seems that double `//` makes the pattern change for all the occurrences.